pom.xml

```diff
@@ -208,7 +208,7 @@
         <dependency>
             <groupId>au.com.dius</groupId>
             <artifactId>pact-jvm-provider-junit_2.11</artifactId>
-            <version>3.3.1</version>
+            <version>3.3.7</version>
```

Comment:
There does not seem to be a compelling reason to go to 3.3.8 right now, so lets do this separately.